node/pkg/driver/node.go

```diff
@@ -475,14 +475,24 @@ func (d *NodeService) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoReque
 	logger.Debugf(">>>> NodeGetInfo: called with args %+v", *req)
 	defer logger.Debugf("<<<< NodeGetInfo")
 
-	iscsiIQN, err := d.NodeUtils.ParseIscsiInitiators("/etc/iscsi/initiatorname.iscsi")
-	if err != nil {
-		return nil, status.Error(codes.Internal, err.Error())
+	iscsiIQN, iscsi_err := d.NodeUtils.ParseIscsiInitiators("/etc/iscsi/initiatorname.iscsi")
+	fcList, fc_err := d.NodeUtils.ParseFCPortsName("/sys/class/fc_host/host*/port_name")
+
+	if iscsi_err != nil && fc_err != nil {
+		errMessage := "errs when get iqn: " + iscsi_err.Error() + ";err when get FC ports: " + fc_err.Error()
+		return nil, status.Error(codes.Internal, errMessage)
+	}
+
+	if iscsi_err != nil {
+		logger.Debugf("when get iqn from node %v, got the error %v", d.Hostname, iscsi_err.Error())
```

Comment:
why its debug and not error or warning?
how iscsi_err can accrue?

sounds like hiding error